low-level/files/opt/seagate/sspl/setup/sspl_config.py

```diff
@@ -49,7 +49,8 @@ def __init__(self):
         """Init method for sspl setup config."""
         self._script_dir = os.path.dirname(os.path.abspath(__file__))
         self.role = None
-        Conf.load('sspl', 'yaml://%s' % consts.file_store_config_path)
+        Conf.load('sspl', consts.sspl_config_path)
+        Conf.load('sspl_test', consts.sspl_test_config_path)
```

Comment:
why sspl_test config being loaded here. All test specific code & config needs to be sspl test specific code only

Reply:
As part of config, we call conf_based_sensor_enable.py script to overwrite sensor information in all config.
So its loaded here and used inside that script.
I look forward your comment on keeping this here.

Reply:
I will take this to sspl_test.

Reply:
Done